ts/src/bitget.ts

```diff
@@ -2241,28 +2238,51 @@ export default class bitget extends Exchange {
         if (limit !== undefined) {
             request['limit'] = limit;
         }
+        if (since !== undefined) {
+            request['startTime'] = since;
+        }
         const extended = this.extend (request, params);
         let response = undefined;
         if (market['spot']) {
-            response = await this.publicSpotGetMarketFills (extended);
+            response = await this.publicSpotGetMarketFillsHistory (extended);
```

Comment:
@Dan-krm when there's different methods capable of doing it, I think we should do something like this: https://github.com/ccxt/ccxt/pull/18679 (make them available through an option)